flake.nix

```
@@ -37,10 +34,10 @@
             installPhase = ''
               runHook preInstall
 
-              DESTDIR="$out" PREFIX="/usr" make install
+              DESTDIR="$out" PREFIX= make install
 
-              wrapProgram "$out/usr/bin/xdg-ninja" \
-                --prefix PATH : "${pkgs.lib.makeBinPath [ pkgs.glow pkgs.jq ]}"
+              wrapProgram "$out/bin/xdg-ninja" \
+                --prefix PATH : "${pkgs.lib.makeBinPath (with pkgs; [ glow jq ])}"
```

Because of #271 we also need to include find here:
```suggestion
                --prefix PATH : "${pkgs.lib.makeBinPath (with pkgs; [ glow jq findutils ])}"
```